test/aave-v3/setup/TestSetupVaults.sol

```diff
@@ -103,6 +103,10 @@ contract TestSetupVaults is TestSetup {
             );
         }
 
+        supplier1 = suppliers[0];
+        supplier2 = suppliers[1];
+        supplier3 = suppliers[2];
+
         vaultSupplier1 = VaultUser(payable(suppliers[0]));
         vaultSupplier2 = VaultUser(payable(suppliers[1]));
         vaultSupplier3 = VaultUser(payable(suppliers[2]));
```

Comment:
then `address(supplier1) == address(vaultSupplier1)` right?

I don't the other changes are then necessary

Reply:
Ok but then I'd rather use `supplier1` or `vaultSupplier1` everywhere, exclusively
I think it's a matter of another PR though?

Reply:
Not sure because the use of `vaultSupplier` is in this PR, perhaps it can be rolled back?

Reply:
the vaultSupplier is already used in the compound tests. I will reconcile this with your suggestion and have this PR revert to using supplier1 only.

Reply:
vaultSupplier is necessary to use in this repo because of the vault user's exclusive functions. So I figure we would like to use vaultSupplier everywhere in the vault tests, and not supplier. This is the change I made, so not sure if we want to do anything different?

Reply:
Also, the reason that supplier is even assigned is to make sure that no mistake would be made in interchanging between the two, as has been done previous to this PR.

Reply:
Oh `vaultSupplier` is used everywhere now? If yes then lgtm